environment.yml

```diff
@@ -0,0 +1,9 @@
+name: jacques
+channels:
+    -  defaults
+dependencies:
+    - python=3.11
+    - numpy
+    - pandas
+    - tensorflow
+    - tensorflow_probability
```

Comment:
Noting that in order to get the demo notebook to run, I also had to install matplotlib and ipympl.  However:

- these are not really dependencies of the package
- I'm feeling down on conda lately.  In the flusion project, I ran into problems that I could not solve with specifying package versions for things installed using conda and using pip.
- we should probably move toward using a more modern setup with `pyproject.toml` file instead of a `setup.py` file.  As part of that, we could set up a block with requirements for running the demo and requirements for doing package development.

So I guess I'm leaning toward just not necessarily solving these problems right now, and using a non-conda based solution that addresses all of these points together.  I don't have clear in my head exactly what that solution looks like though.